tests/test_parametric_reactors/test_SingleNullBallReactor.py

```diff
@@ -0,0 +1,102 @@
+
```

Comment:
**DEAD CODE**

When leaving large amounts of dead code in the repository, it helps to add a `TODO` comment to explain why it is there as they tend to linger for extended periods of time.